ansible/README.md

```diff
@@ -0,0 +1,121 @@
+Deploying Openwhisk using Ansible
+=========
+
+
+### Getting started
+
+If you want to deploy Openwhisk locally using Ansible, you first need to install Ansible on your development environment:
+
+#### Ubuntu users
+```
+sudo apt-get install python-pip
+sudo pip install ansible==2.0.2.0
+```
+
+#### Vagrant users
+Nothing to be done, Ansible is already installed during vagrant provisioning.
+
+#### Mac users
+It is assumed that a VM has been provisioned using **docker-machine**.
+
+```
+brew install python
+pip install ansible==2.0.2.0
+
+cd ansible
+ansible-playbook setup.yml
```

Comment:
`whisk` as the docker-machine name as default. If user has another machine pls do

```
ansible-playbook setup.yml -e docker_machine_name=anotherwhisk
```

The task will use `docker-machine` command to find out the IP of the docker-machine. In case, you provided an invalid docker-machine name  (e.g. typo), you will see an error by running ansible.